arlo-client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx

```diff
@@ -202,16 +196,19 @@ describe('Audit Setup > Review & Launch', () => {
         dummyJurisdictions
       )
     )
-    const { findByText } = await asyncActRender(
+    const { findByText, getAllByText } = await asyncActRender(
       <Router>
         <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
       </Router>
     )
     const launchButton = await findByText('Launch Audit')
     fireEvent.click(launchButton, { bubbles: true })
+    await findByText('Are you sure you want to launch the audit?')
```

Comment:
Huzzah, you figured it out! This was the bit I was stuck on. Well done!

Reply:
Yeah I was staring at it for a while like 🤔